doc/source/user_guide/groupby.rst

```diff
@@ -537,41 +574,84 @@ Another aggregation example is to compute the number of unique values of each gr
 
 .. note::
 
-   Aggregation functions **will not** return the groups that you are aggregating over
+   Aggregation functions **will not** operate on the groups that you are aggregating over
    if they are named *columns*, when ``as_index=True``, the default. The grouped columns will
```

Comment:
_which is_ the default

Reply:
The original and the modified version are incorrect and confusing, respectively; I'll clean this up.

Reply:
I agree.